### PR TITLE
The max size of a key in LMDB is actually 511, not 512.

### PR DESCRIPTION
**Pull request by dureuill**
# Pull Request

The documentation currently informs users of limitations on the size of various data to 512 bytes, due to the maximum size of a LMDB key.

That maximum size is actually 511, as evidenced by https://github.com/meilisearch/meilisearch/issues/5050.

This documentation PR indicates the correct value for all involved values, while the companion code PR (https://github.com/meilisearch/meilisearch/pull/5144) replaces an internal error by a user error by checking against the correct value.

### CODE REVIEW

**Review by guimachiavelli — APPROVED** [review]
LGTM! Should I wait until the other PR you mention has been merged?

**Comment by dureuill** [comment]
hello, not necessarily, while the error message is inaccurate in v1.11, the behavior was already as described in this documentation PR.

**Pull request merged**
